apps/ledger-live-desktop/src/renderer/animations/index.tsx

```diff
@@ -22,8 +22,10 @@ const Animation = ({
   rendererSettings?: LottieProps["options"]["rendererSettings"];
   isPaused?: boolean;
   isStopped?: boolean;
-}) =>
-  animation ? (
+}) => {
+  // in case of playwright tests, we want to completely stop the animation
+  const isPlaywright = !!getEnv("PLAYWRIGHT_RUN");
+  return animation ? (
```

Comment:
Here the component changes its behavior for tests, can we find another way to do it? I think that we will have better separation if components are not aware of the external world.
A solution I have in mind is to mock the Lottie component that returns the Lottie component itself with enforced autoplay: false
`LottieMock = props => <Lottie {...props, options: {...props.options, autoplay: false}} />`

That way we don't change our code for playwright + if someone uses the Lottie component elsewhere we don't break screenshots. wdyt?

Reply:
I agree with you and it's indeed a bad pattern we use not just here.
it will require more time to come up to a proper solution tho, we'll need to iterate on this.
are you suggesting that we then introduce another env (other than PLAYWRIGHT_RUN) to trigger the LottieMock mode?

Reply:
Thanks for the explanation, I was not aware that we can't mock modules with playwright.
Then the solution you propose is the best one.
The other solution we have is to use the mask but it will require updating every test that uses a page with a Lottie file + we will lose the info of which Lottie file is used on the screenshot, so it's a poor solution.